ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -633,9 +634,17 @@ export const itwEidIssuanceMachine = setup({
                 "This state handles the CIE preparation screen, where the user is informed about the CIE card",
               entry: "navigateToCiePreparationScreen",
               on: {
-                next: {
-                  target: "StartingCieAuthFlow"
-                },
+                next: [
+                  {
+                    guard: "isL3FeaturesEnabled",
+                    actions: "navigateToCieReadCardL3Screen",
+                    target: "StartingCieAuthFlow"
+                  },
+                  {
+                    actions: "navigateToCieReadCardL2Screen",
```

Comment:
Do we plan to dismiss this screen with the older card reader? I wouldn't want to keep two card reader screens in the long run.